### PR TITLE
🎨 Backend part of UX improvements

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

Removes the "changing" part of the message going to the frontend, so that @odeimaiz can change how copying is shown.

Maybe when time is right we shall better go with https://github.com/ITISFoundation/osparc-simcore/pull/6288 but for now this appears to critical to apply it.

## Related issue/s
- relates to https://github.com/ITISFoundation/osparc-issues/issues/1628

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by pcrespov — APPROVED** [review]
:boom:

File: services/storage/src/simcore_service_storage/s3_utils.py [hunk 1]
```diff
@@ -39,8 +39,7 @@ def _update(self) -> None:
         update_task_progress(
             self.task_progress,
             f"{self.task_progress_message_prefix} - "
-            f"{parse_obj_as(ByteSize,self._total_bytes_copied).human_readable()}"
-            f"/{self.total_bytes_to_transfer.human_readable()}]",
+            f"{self.total_bytes_to_transfer.human_readable()}]",
```
Comment (odeimaiz) [inline services/storage/src/simcore_service_storage/s3_utils.py:42]: Do we still need that last ``]``?

**Review by odeimaiz — APPROVED** [review]
Thanks

File: services/storage/src/simcore_service_storage/s3_utils.py (hunk 1 above)
Comment (sanderegg) [reply services/storage/src/simcore_service_storage/s3_utils.py:42]: probably not indeed

**Pull request merged**